assets/cubyz/biomes/desert/oasis/_defaults.zig.zon

```diff
@@ -0,0 +1,15 @@
+.{
+	.properties = .{
+		.hot,
+		.dry,
+	},
+	.validPlayerSpawn = false,
```

Comment:
it has wood, I think it should be a valid player spawn (at least the outer biome, not nexessarily the pond).